tools/sim/bridge/metadrive.py

```diff
@@ -0,0 +1,142 @@
+import math
+import numpy as np
+import time
+
+from openpilot.tools.sim.bridge.common import World, SimulatorBridge
+from openpilot.tools.sim.lib.common import vec3, SimulatorState
+from openpilot.tools.sim.lib.camerad import W, H
+
+class MetaDriveWorld(World):
+  def __init__(self, env, ticks_per_frame: float, dual_camera = False):
+    super().__init__(dual_camera)
+    self.env = env
+    self.ticks_per_frame = ticks_per_frame
+    self.dual_camera = dual_camera
+
+    self.steer_ratio = 15
+
+    self.vc = [0.0,0.0]
+
+    self.reset_time = 0
+
+  def get_cam_as_rgb(self, cam):
+    cam = self.env.engine.sensors[cam]
+    img = cam.perceive(self.env.vehicle, clip=False)
+    if type(img) != np.ndarray:
```

Comment:
why would the type change?

Reply:
if you are using cuda, you get a cupy array instead which is on the GPU, and needs to be moved